package.json

```diff
@@ -2,7 +2,7 @@
   "name": "react-native-safari-view",
   "version": "0.1.0",
   "description": "A React Native wrapper for Safari View Controller",
-  "main": "SafariViewMananger.ios.js",
+  "main": "index.ios.js",
   "scripts": {
```

Comment:
This looks weird to me, since the repo doesn't have an `index.ios.js` file. 

After reading the issue you liked to, it seems more like a react-native bug.